bindings_flutter/src/libxmtp_api.rs

```diff
@@ -137,6 +137,13 @@ impl Client {
         });
     }
 
+    pub async fn is_active_group(&self, group_id: Vec<u8>) -> Result<bool, XmtpError> {
+        self.inner.sync_welcomes().await?;
```

Comment:
Wondering if this is needed, here and in the other methods? I don't think the caller would be able to get a group_id without syncing the group it belongs to. Also, in bindings_ffi, the integrator can control when the syncing happens, via FfiConversations.sync()

Reply:
My tests hit this when clients B and C made `is_active_group` calls against a `groupId` the test held onto from client A's creation call. But you're right, in a real situation they would only have access to the `groupId` because they got it from an earlier sync'd list/stream call. I'll ✂️